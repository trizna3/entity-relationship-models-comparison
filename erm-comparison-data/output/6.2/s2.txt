tickets-tickets
screenings-screening
films-films
customers-users
auditoriums-cinema halls
seats-EMPTY
