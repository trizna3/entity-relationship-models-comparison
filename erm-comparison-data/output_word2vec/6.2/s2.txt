auditoriums-cinema
screenings-screening
tickets-tickets
customers-users
seats-EMPTY
films-films
