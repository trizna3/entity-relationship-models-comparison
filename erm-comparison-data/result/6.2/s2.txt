Mapping 5/6 pairs, 1 additional pairs.
Missing pairs: 
 seats-seats layout
Additional pairs: 
 seats-empty

Mapping  0/0 transformations, 0 additional transformations.
Missing transformations: -
Additional transformations: -
