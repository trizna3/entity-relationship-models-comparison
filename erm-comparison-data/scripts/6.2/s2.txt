
entity set cinema {
};

entity set "cinema halls" {
};


entity set screening {
	"date"
	"time"
};

entity set films {
	"name"
};

entity set tickets {
	"row"
	"column"
};

entity set users {
};

entity set "seats layout" {
	"rows"
	"columns"
};


association between 
{
	cinema ONE 
	"cinema halls" MANY
}{};

association "has license" between 
{
	cinema MANY 
	films MANY
}{};

association between
{
	cinema ONE
	screening MANY
}{};

association between 
{
	screening MANY 
	"cinema halls" ONE
}{};

association between 
{
	screening ONE 
	tickets ONE
}{};

association between 
{
	tickets MANY
	users ONE
}{};

association between 
{
	"cinema halls" MANY
	"seats layout" MANY
}{};