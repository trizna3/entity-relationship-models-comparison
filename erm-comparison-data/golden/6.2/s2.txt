auditoriums-cinema halls
screenings-screening
films-films
customers-users
tickets-tickets
seats-seats layout